### PR TITLE
Feature finance report

### PR DESCRIPTION
**Pull request by Kajochi**

### CODE REVIEW

File: backend/src/main/java/de/neuefische/capstone/backend/financereport/FinanceReportCalculate.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    private BigDecimal calculateIntervalNum(Interval entryInterval, Interval reportPeriod) {
+        int entryMultiplier = getMultiplier(entryInterval);
+        int reportMultiplier = getMultiplier(reportPeriod);
+
+        return BigDecimal.valueOf(reportMultiplier).divide(BigDecimal.valueOf(entryMultiplier), 3, RoundingMode.HALF_DOWN);
+
+    }
+
+}
```
Comment (saman1357) [inline backend/src/main/java/de/neuefische/capstone/backend/financereport/FinanceReportCalculate.java]: missing empty line at the end of the file 😱

Comment (Kajochi) [reply backend/src/main/java/de/neuefische/capstone/backend/financereport/FinanceReportCalculate.java]: God damn it. Thank you :)

File: backend/src/main/java/de/neuefische/capstone/backend/model/Interval.java [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+
     MONTHLY,
     QUARTERLY,
     HALF_YEARLY,
     YEARLY,
-    ONCE
+    ONCE;
+
+
+    public static int getMultiplier(Interval interval) {
```
Comment (saman1357) [inline backend/src/main/java/de/neuefische/capstone/backend/model/Interval.java:12]: nice 👌

File: backend/src/test/java/de/neuefische/capstone/backend/financereport/FinanceReportServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                        new BigDecimal("300.000")
+                )
+        );
+        List<FinanceReport> actual = financeReportService.getFinanceReports();
+
+        //THEN
+        assertEquals(expected, actual);
+    }
+
+}
```
Comment (saman1357) [inline backend/src/test/java/de/neuefische/capstone/backend/financereport/FinanceReportServiceTest.java:98]: missing empty line at end of file

File: frontend/src/hooks/useStore.ts [hunk 4]
```diff
@@ -4,10 +4,13 @@ import {Entry} from "../model/Entry.ts";
 import {EntryWithNoId} from "../model/EntryWithNoId.ts";
 import {Simulate} from "react-dom/test-utils";
 import error = Simulate.error;
+import {FinanceReport} from "../model/FinanceReport.ts";
 
 type State = {
```
Comment (saman1357) [inline frontend/src/hooks/useStore.ts:9]: nice 👍

File: frontend/src/model/FinanceReport.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+export type FinanceReport = {
+
+    period: Interval,
+    totalIncome: string,
+    totalExpenses: string,
+    fixCosts: string,
+    variableCosts: string,
+    balance: string,
+
+}
```
Comment (saman1357) [inline frontend/src/model/FinanceReport.ts]: empty line...

**Review by saman1357 — APPROVED** [review]
LGTM 👌

**Pull request merged**